pyproject.toml

```
@@ -65,14 +66,16 @@ dependencies = [
     "SQLAlchemy-Utils==0.40.0",
     "typer==0.7.0",
     "uvicorn[standard]~=0.20.0",
-    "nwa-stdlib~=1.4.6",
-    "oauth2-lib~=1.2.5",
+#    "nwa-stdlib~=1.4.6",
+    "nwa-stdlib @ git+https://github.com/workfloworchestrator/nwa-stdlib@python39-compatibility#egg=nwa-stdlib",
+#    "oauth2-lib~=1.2.5",
+    "oauth2-lib @ git+https://github.com/workfloworchestrator/oauth2-lib@python39-compatibility#egg=oauth2-lib",
     "markupsafe==2.0.1",
     "bandit==1.7.2",
     "tabulate==0.9.0"
 ]
 description-file = "README.md"
-requires-python = ">3.9,<3.12"
+requires-python = ">3.8,<3.12"
```

I would prefer showing the required python version then above a previous version
```suggestion
requires-python = ">=3.9,<3.12"
```

---

:+1: done